core/templates/services/UpgradedServices.ts

```diff
@@ -69,6 +69,12 @@ import { ConceptCardObjectFactory } from
 import { ContextService } from 'services/context.service';
 import { ContinueValidationService } from
   'interactions/Continue/directives/continue-validation.service';
+import { ContributionOpportunitiesService }
+// eslint-disable-next-line max-len
+  from 'pages/community-dashboard-page/services/contribution-opportunities.service';
+import { ContributionOpportunitiesBackendApiService }
```

Comment:
Add this in sorted order

Reply:
@h-ARTS, is this comment resolved?

Reply:
@DubeySandeep This is resolved, PTAL!

Reply:
Thanks for resolving the comment @h-ARTS. In general, make sure to reply to the comments which are resolved [as suggested in the wiki page](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#address-review-comments-until-all-reviewers-give-lgtm-looks-good-to-me).